src/CandidateList.js

```diff
@@ -0,0 +1,139 @@
+import React, {useState, useEffect} from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+import CandidateCard from './CandidateCard';
+import { useCookies } from 'react-cookie';
+import LoginPage from './LoginPage';
+import SignUpPage from './SignUpPage';
+const dataURL = 'http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:8080/candidates';
+function CandidateList() {
+    const [candidates, setCandidates] = useState(null);
+    const [isLogIn, setIsLogIn] = useState(false);
```

Comment:
isLoggedIn이 더 적합해보여용
```suggestion
    const [isLoggedIn, setIsLoggedIn] = useState(false);
```